### DISCUSSION TITLE
Is this correct way to limit page load timeout for LoadUrlAsync ?

### DISCUSSION THREAD

**Discussion by assopri** [cefsharp/CefSharp#3729] [General]
Hi! Just wanted to know it is correct way to limit page loading timeout?

`Task<LoadUrlAsyncResponse> task = Browser.LoadUrlAsync(url);
            task.Wait(TimeSpan.FromMilliseconds(LoadTimeoutMs));
            if(!task.IsCompleted) throw new TimeoutException("Page is loading to long");`

**Comment by amaitland** [comment]
As you are using Task.Wait it's whatever works for you.

**Reply by assopri** [reply]
THank you!
